adoc/Release-Notes.adoc

```diff
@@ -21,6 +21,67 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_2_1']
+=== Release 1.2.1, September 2018
+
+[id='sec.1_2_1.issue']
+==== Known Issues
+
+* If you are upgrading to {product} 1.2.1, you will need to run the `helm upgrade` command with two additional parameters: `helm upgrade --force --recreate-pods`. This will be the case with all future 1.x upgrades due to the expected behaviour with Kubernetes that unready pods won’t get upgraded.
+
+* Similar to {caaspa} 3, {azure} uses an updated version of Kubernetes that mandates a stricter security policy via PodSecurityPolicy (PSP), which is included as part the {product} Deployment Guide. Any namespace tied to {product} requires privileged ports to be accessible needs to have to have a PSP set appropriately for access. This would include the default conventions of `scf`, `uaa`, `stratos-ui`, `mysql-sidecar` and `postgres-sidecar` as per our documentation.
```

Comment:
`as per our documentation`... -> Can you add a link to a specific section instead?

Reply:
Added link to CaaSP 3 documentation, docs ticket filed to update Azure/AKS documentation then we can consider updating link. The copy used matches what was recorded in 1.2.0.